flink-table/flink-sql-client/src/test/resources/sql/select.q

```diff
@@ -221,3 +221,72 @@ FROM (VALUES
 +----------+---------------------+-------------------------+-------------------------------+-------------------------+-------------------------+-------------------------------+
 2 rows in set
 !ok
+
+# ==========================================================================
+# Testing behavior of sql-client.display.max-column-width
```

Comment:
Add the tests in the above `sql-client.display.max-column-width` section? We don't need to re-register `testUserData`. Besides, please verify `table.display.max-column-width` should work. It would be better to replace all the `sql-client.display.max-column-width` with `table.display.max-column-width`, and just add several tests for compatibility of `sql-client.display.max-column-width`.

Reply:
Do you mean the part starting from line 119? It is for `streaming`. This part is for `batch`

Reply:
It is really difficult to use cross modules ConfigOptions and keep them synchronized. There are options:

- option 1: use `SqlConfigOption#DISPLAY_MAX_COLUMN_WIDTH` as the fallback key of `TableConfigOption#DISPLAY_MAX_COLUMN_WIDTH`. This does not work because `flink-table-api-java` does not depends on `flink-sql`. Adding new dependency will trigger a cycle dependency issue.
- option 2: use `TableConfigOption#DISPLAY_MAX_COLUMN_WIDTH` as the fallback key of `SQLConfigOption#DISPLAY_MAX_COLUMN_WIDTH`. If ResultDescriptor uses `TableConfigOption#DISPLAY_MAX_COLUMN_WIDTH`, any change of `sql-client.display.max-column-width` will not be able to work.

Reply:
The reason of the split is that I didn't find a way to remove it, once sql-client.display.max-column-width has been set, which make it impossible to test  table.display.max-column-width again.